src/components/live/home.tsx

```diff
@@ -79,21 +86,25 @@ const QuickLinks = () => {
         <div className="mt-5 w-full border-t-2 border-primary-400 md:w-[120%]" />
         <QLink
           text="Event Guide"
-          link="#"
+          link="https://island-denim-3d7.notion.site/Hack-Western-11-Hacker-Package-142e351b228b8030a3deebfa557063ec"
           image="/images/icons/arrowupright.svg"
         />
         <div className="w-full border-t-2 border-primary-400 md:w-[120%]" />
-        <QLink text="Join our Slack" link="#" image="/images/icons/slack.svg" />
-        <div className="w-full border-t-2 border-primary-400 md:w-[120%]" />
+        <QLink
+          text="Join our Slack"
+          link="https://join.slack.com/t/hackwestern11/shared_invite/zt-2unzxw9iq-xS9AYyz71BlYkGW7VdAceg"
+          image="/images/icons/slack.svg"
+        />
+        {/*<div className="w-full border-t-2 border-primary-400 md:w-[120%]" />
         <QLink
           text="Add Schedule to GCal"
           link="#"
           image="/images/icons/gcal.svg"
-        />
+        />*/}
```

Comment:
here's the link sir: https://calendar.google.com/calendar/embed?src=c7677dbb10f476721982c9755440429fa1791b6818969805682b81e2eaf2bbb7%40group.calendar.google.com&ctz=America%2FToronto